### DISCUSSION TITLE
Toolkit and getting started for ATproto community hack sessions

### DISCUSSION THREAD

**Discussion by bmann** [ATProtoApps/community#11] [General]
What kind of toolkit, getting started, lexicon, API best practices can we provide to support community hack events?

parcon mentions a civic hacknight with prizes https://bsky.app/profile/patcon.bsky.social/post/3lcoumpxcwc2v

**Comment by patcon** [comment]
Good guy, that parcon (Commenting for cleverisms, but mostly to sub)

---

EDIT: I'm a bit broke and so need to focus on getting a gig for myself before I sign myself up for anything substantial for this.

BUT I could maybe sketch out what a kit might look like. Thinking a one-pager that anyone could bring to their hacknight organizers to either encourage them to run something, or to be allowed to run it as a service-oriented project (a project to run a competition to start other projects is the sort of polycentric organizing that gives me fuzzies 😁 )
